### ISSUE TITLE
什么时候支持full训练的seq_cls

### ISSUE THREAD

**Issue by QixingJiang** [modelscope/ms-swift#3063]
目前我使用最新的swift Version: 3.1.0.dev0
很高兴看到能支持序列分类seq_cls任务
然而我发现例如使用Qwen2vl模型进行多模态的序列分类任务训练时，目前仅能支持lora方式的训练，如同提供的example那样：examples/train/seq_cls/qwen2_vl/sft.sh 
当我尝试使用full进行训练而不是lora时，训练正常，推理的结果会存在问题，所有输出都为同一个标签，目前应该是暂未支持full训练？如果是的话，请问什么时候能支持？

**Comment by Jintao-Huang** [comment]
支持全参数啊，可以看看如何推理不

**Comment by QixingJiang** [comment]
> 支持全参数啊，可以看看如何推理不

感谢回复，推理的代码：
CUDA_VISIBLE_DEVICES=0 \
MAX_PIXELS=602112 \
swift infer \
     --adapters /path/to/ckpt \
     --val_dataset /path/to/val_dataset.jsonl
full训练的代码：
MAX_PIXELS=602112 CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 NPROC_PER_NODE=8 swift sft \
    --model /path/to/local/qwenvl2_instruct_ckpt \
    --model_type qwen2_vl \
    --train_type full \
    --dataset /path/to/train_dataset.jsonl \
    --val_dataset /path/to/val_dataset.jsonl \
    --dataloader_num_workers 4 \
    --num_labels 4 \
    --task_type seq_cls \
    --use_chat_template true \
    --deepspeed zero2 \
    --attn_impl flash_attn \
我用这种方式full训练后，保存下来的last_ckpt和中间的ckpt进行推理的时候，不管什么图（label可能是0,1,2,3都有），输出的label都是1，很奇怪。
数据格式为自定义的数据集：
{"messages": [{"role": "user", "content": "<image>分类这张图。"}], "images": ["/path/to/image1.jpg"], "label": 0}
然后我用lora训练后（lora训练代码基本沿用examples/train/seq_cls/qwen2_vl/sft.sh），推理是正常的，结果符合预期。推理的代码跟上面一样。

**Comment by Jintao-Huang** [comment]
全参数使用--model xxx

**Comment by QixingJiang** [comment]
> 全参数使用--model xxx

感谢解答，实测没问题了，确实是我自己使用方式有误，该issue将关闭~

**Comment by kike-0304** [comment]
> > 全参数使用--model xxx
> 
> 感谢解答，实测没问题了，确实是我自己使用方式有误，该issue将关闭~

怎么解决的呀

**Comment by QixingJiang** [comment]
> > > 全参数使用--model xxx
> > 
> > 
> > 感谢解答，实测没问题了，确实是我自己使用方式有误，该issue将关闭~
> 
> 怎么解决的呀

就是把--adapter换成--model进行推理 其余参数不变 
虽然不太确定adapter和model底层实现方式有什么区别, 这一块文档也没有具体btw, 最近发现一个很奇怪的bug,我用InternVL2.5-1B进行全参数SFT后用--model 输出都是空白的"", --adapter会有正常输出...

**Comment by waltonfuture** [comment]
> --val_dataset /path/to/val_dataset.jsonl

你好！请问这里面--val_dataset /path/to/val_dataset.jsonl 的数据格式应该是什么样的呀

**Comment by waltonfuture** [comment]
> > --val_dataset /path/to/val_dataset.jsonl
> 
> 你好！请问这里面--val_dataset /path/to/val_dataset.jsonl 的数据格式应该是什么样的呀

打扰了，已解决~
